structure-proposal.md

```diff
@@ -0,0 +1,104 @@
+# Modular Base-Structure for the Simulation Study
+
+## Goal
+
+The primary goal of this project is to develop a new (adversarial) collaboration approach for simulation studies, aiming to bolster their generalizability by delegating critical, subjectivity-prone decision-making processes when conducting a simulation based study to neutral or adversarial parties. To establish a foundational framework within which such an adversarial collaborative approach can be developed and showcased the following aims to identify the structure of all decision-making elements that can be subjected to such collaboration in simulation studies. 
+
+Consequently each individual step of the foundational structure should be:
+
+1) sufficiently modular to allow for independent evaluation and modification of critical decision-making processes by neutral or adversarial collaborators,
+
+2) narrow and clear enough to ensure an outcome that is compatible with a set of (predefined) evaluation criteria and (optionally) that can be compared post hoc to a parallel non-collaborative simulation.
+
+The overall set of steps of the simulation should be comprehensive and complete to ensure all relevant and only the relevant steps for either 1) or 2) are included.
```

Comment:
I don't quite understand what this means

Reply:
I think this goal it mostly coincides with your proposed goal. The aim is to have a structure that lists all steps that can be modulated in some way to maximize the use of collaboration and all steps that can be standardized to maximize comparability.